metadata-transformer/src/test/java/no/unit/nva/metadata/service/MetadataServiceTest.java

```diff
@@ -111,10 +116,26 @@
     public static final String SECOND_ISBN_ISBN13_VARIANT = "9781627050128";
     public static final String ONLINE_ISSN = "2052-2916";
     public static final String PRINT_ISSN = "0969-0700";
-    public static final String ENSURE_ONLY_FILTERING_TITLES = "McLongName, Longy (Colonel (ret'd))";
     private static final TestAppender logger = LogUtils.getTestingAppenderForRootLogger();
     private WireMockServer wireMockServer;
 
+    @Test
+    public void getJournalIdFromPublicationChannelsByGivenJournalName()
+            throws IOException, InterruptedException {
+        var responseBody = IoUtils.stringFromResources(Path.of("publication_channels_442850_response.json"));
+        var httpResonse = mock(HttpResponse.class);
+        when(httpResonse.statusCode()).thenReturn(SC_OK);
+        when(httpResonse.body()).thenReturn(responseBody);
+        var httpClient = mock(HttpClient.class);
+        when(httpClient.send(any(HttpRequest.class), any())).thenReturn(httpResonse);
+        var metadataService = new MetadataService(httpClient);
+        var name = "Edinburgh Journal of Botany";
+        var year = 2003;
```

Comment:
Weird that the year doesn't match the year in the. URI

Reply:
right, was weird